extensions/interactions/MusicNotesInput/directives/oppia-interactive-music-notes-input.directive.ts

```diff
@@ -277,16 +277,12 @@ angular.module('oppia').directive('oppiaInteractiveMusicNotesInput', [
           var validNoteArea = element.find(
             '.oppia-music-input-valid-note-area');
           for (var i = 0; i < NOTE_TYPES.length; i++) {
-            var innerDiv = $('<div></div>')
-              .data('noteType', NOTE_TYPES[i])
-              .addClass(function(index, currentClassName) {
-                var addedClass = null;
-                if ($(this).data('noteType') === NOTE_TYPE_NATURAL) {
-                  addedClass = 'oppia-music-input-natural-note';
-                  $(this).addClass('oppia-music-input-natural-note');
-                }
-                return addedClass;
-              });
+            var addedClass = null;
+            if (NOTE_TYPES[i] === NOTE_TYPE_NATURAL) {
+              addedClass = 'oppia-music-input-natural-note';
+            }
+            var innerDiv = $(`<div class="${addedClass}"></div>`)
+              .data('noteType', NOTE_TYPES[i]);
```

Comment:
Can we preferably not use jQuery at all, our long term goal is to get rid of it from the codebase.

Reply:
ditto elsewhere

Reply:
Yes, I agree that would be ideal and I believe that is the ultimate goal which is being reached by performing small steps, including this effort of removing the addClass/removeClass function calls: #11329

There is already a PR migrating the whole oppia-interactive-music-notes-input directive to Angular and removing jquery: #14841

So we might close this one in favor of that PR, or accept this PR as a half-solution. @vojtechjelinek wdyt?

Thanks!

Reply:
Actually, #14841 is not removing jQuery, it just adds TODOs. So your PR still makes sense to do.

Reply:
ok! I will get my code merged with develop and update this PR